src/js/languages/default.json

```
@@ -104,6 +104,10 @@
     "openMenu": "Open Menu",
     "closeMenu": "Close Menu"
   },
+  "meter": {
+    "singular": "{type} meter with value {meterValue} out of {max}",
+    "plural": "{type} meter with values {meterValue} out of {max}"
```

One thing I just realized is the `type` isn't currently localizable. We should have keys in here for type. Although a bit wordy, I think a more flexible thing for localizers would be:
```suggestion
    "bar": {
      "singular": "Bar meter with value {meterValue} out of {max}",
      "plural": "Bar meter with values {meterValue} out of {max}"
    },
    "circle": {
      "singular": "Circle meter with value {meterValue} out of {max}",
      "plural": "Circle meter with values {meterValue} out of {max}"
    }
```

---

aw okay good point I can restructure

---

@MikeKingdom updated structure